client/blocks/login/index.jsx

```diff
@@ -260,7 +273,7 @@ class Login extends Component {
 
 	render() {
 		return (
-			<div>
+			<div className={ classname( 'login', { 'login--jetpack': this.props.jetpack } ) }>
```

Comment:
Should the classname be `login__jetpack` instead?

Reply:
BEM is `block[__element][--modifier]`, which is used around Calypso sometimes and looks similar to the enforced namespaces. `jetpack` here is a modifier, it's a different styling of the `login` block. The `--` was intentional.

However, you've inspired me to look at the [CSS guidelines](https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/css.md#csssass-coding-guidelines):

> This is an example of a declaration:
>
> `.component__element.is-modifier {}`
>
> Two important considerations:
>
> * The `component` fragment matches the folder name of the React component it's providing styles to.
> * The `.is-modifier` class should never be written on its own outside of the component class. This avoids naming collisions and provides a consistent use of modifiers.

I'll change this to an `.is-jetpack` modifier 👍 0338d18aeb6ff9d85bfc451708bc86b207e1a5a2